code/game/machinery/computer3/computers/HolodeckControl.dm

```diff
@@ -90,11 +90,11 @@
 			if(target)
 				loadProgram(target)
 
-		else if("burntest" in href_list)
-			if(!emagged)	return
-			target = locate(/area/holodeck/source_burntest)
-			if(target)
-				loadProgram(target)
+//		else if("burntest" in href_list)
+//			if(!emagged)	return
+//			target = locate(/area/holodeck/source_burntest)
+//			if(target)
+//				loadProgram(target)
```

Comment:
This is something I accidentally left in while I was testing solutions to the holodeck fire being overpowered, I probably re-added it and saved on the same version.